usage/camera/raspicam/v4l2.md

```diff
@@ -1,13 +1,29 @@
 ## Using the V4L2 driver
 
-The V4L2 driver provides a Linux standard driver for accessing camera features. This is the driver needed to use the Raspberry Pi camera range as, for example, webcams.
+The V4L2 driver provides a standard Linux driver for accessing camera features. This is the driver needed to use the Raspberry Pi camera range as, for example, webcams.
 
-The V4L2 driver provides a standard API over the firmware-based camera system.
+The V4L2 driver provides a standard API on top of the firmware-based camera system.
 
 
 ### Installing the V4L2 driver
 
 Installation of the V4L2 driver is automatic. It is loaded as a child of the VCHIQ driver, and once loaded it will check how many cameras are attached and then create the right number of device nodes.
 
+| \dev\videoX | Default Action |
+|-------------|:--------------:|
+| video10     | Decode |
+| video11     | Encode |
+| video12     | Simple ISP |
+| video13     | Full ISP In |
+| video14     | Full ISP Hi-res Out |
+| video15     | Full ISP Lo-res Out | |
+| video16     | Full ISP statistics |
+| video19     | HEVC Decode |
 
 
+
+### Testing the driver
+
+There are many Linux applications that use the V4L2 API, the kernel maintainers provided test tool is called `Qv4l2` and can be installed from the Raspbian OPS repositories as follows:
```

Comment:
Did you really mean "Raspbian" there? :wink:  (and "OPS" is clearly a typo)

Reply:
Thought I fixed that...

Reply:
Oh yeah, "Raspbian OPS" got fixed to "Raspbian OS" in a later commit, my mistake.
But should "Raspbian OS" be "Raspberry Pi OS" ?

Reply:
Yes, thought I had changed that as well, but obviously not.